src/main/java/com/internet/shop/services/OrderServiceImpl.java

```diff
@@ -6,16 +6,15 @@
 import com.internet.shop.models.Order;
 import com.internet.shop.models.ShoppingCart;
 import java.util.List;
-import java.util.stream.Collectors;
 
 @Service
 public class OrderServiceImpl implements OrderServiceInterface {
```

Comment:
```suggestion
public class OrderServiceImpl implements OrderService {
```
Слово Interface краще не використовувати в назвах інтерфейсів

Reply:
fixed